### PR TITLE
Filling in missing email templates for Magento Company and Magento Company Credit  blocks

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) describe missing email templates for Magento Company and Magento Company Credit  blocks (B2B)

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/marketing/email-template-list.html

### CODE REVIEW

File: src/marketing/email-template-list.md [hunk 1]
```diff
@@ -17,31 +17,31 @@ The list of email templates is organized alphabetically by Magento module.
...
+| Assign Company to Customer | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails <br/>**Field:** Default 'Assign Company to Customer' Email|
+| Company Admin Changed to Member | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails<br/>**Field:** Default 'Company Admin Changed To Member' Email|
+| Company Admin Set Inactive | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails<br/>**Field:** Default 'Customer Status Inactive' Email|
+| Company Registration Request | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:** Email Options - Company Registration<br/>**Field:** Default Company Registration Email|
+| Company Status Active1 | **Page:** **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Company Status Change<br/>**Field:** Default 'Company Status Change To Active 1" Emai|
```
Comment (dobooth) [inline src/marketing/email-template-list.md:25]: There are 5 double "Page:"s here.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/marketing/email-template-list.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/marketing/email-template-list.md]: Thank you @dobooth,
I have fixed this!

Could you please approve?

File: src/marketing/email-template-list.md [hunk 2]
```diff
@@ -17,31 +17,31 @@ The list of email templates is organized alphabetically by Magento module.
...
-| Company Registration Request | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Active1 | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Active2 | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Blocked | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Pending Approval | **Page:** <br/>**Section:** <br/>**Field:** |
-| Company Status Rejected | **Page:** <br/>**Section:** <br/>**Field:** |
-| Customer Status Active | **Page:** <br/>**Section:** <br/>**Field:** |
-| Customer Status Inactive | **Page:** <br/>**Section:** <br/>**Field:** |
-| Sales Representative Assigned to Company | **Page:** <br/>**Section:** <br/>**Field:** |
+| Assign Company Admin | **Page:** Customers > [Company Congiguration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails<br/>**Field:** Default 'Assign Company Admin' Email|
```
Comment (dobooth) [inline src/marketing/email-template-list.md:20]:
```suggestion
| Assign Company Admin | **Page:** Customers > [Company Configuration]({% link customers/company-configuration.md %})<br/>**Section:**Customer-Related Emails<br/>**Field:** Default 'Assign Company Admin' Email|
```
Repeat fix for all.

**Review by dobooth — CHANGES_REQUESTED** [review]

**Review by dobooth — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1199`.

**Pull request merged**